files/default/neutron-ha-tool.py

```diff
@@ -89,11 +91,19 @@ def run(args):
     except KeyError:
         ca = None
 
+    # Allow environment to override config file.  This retains
+    # backwards-compatibility and follows conventional precedence.
+    if os.getenv('OS_PASSWORD'):
+        os_password = os.environ['OS_PASSWORD']
+    else:
+        with open('/etc/neutron/os_password') as f:
```

Comment:
You mean so that we can abort with a more friendly error message, rather than an exception and a stacktrace?  Sounds reasonable to me.  Or do you mean something else?